Clean-up/clean_lambda.py

```diff
@@ -0,0 +1,55 @@
+"A script to remove unsubscribed products from the database"
+
+from os import environ as ENV
+import logging
+import json
+
+from dotenv import load_dotenv
+from psycopg2 import connect
+from psycopg2.extras import RealDictCursor
+from psycopg2.extensions import connection, cursor
+
+
+def get_connection() -> connection:
+    "Establishes a connection with the database"
+    return connect(
+        user=ENV["DB_USER"],
+        password=ENV["DB_PASSWORD"],
+        host=ENV["DB_HOST"],
+        port=ENV["DB_PORT"],
+        database=ENV["DB_NAME"]
+    )
+
+
+def get_cursor(conn: connection) -> cursor:
+    "Returns a cursor for the database"
+    return conn.cursor(cursor_factory=RealDictCursor)
+
+
+def delete_unsubscribed(conn: connection, table: str) -> list[dict]:
+    "Deletes info about the unsubscribed products from the database"
+    with get_cursor(conn) as cur:
+        cur.execute(f"""
+            DELETE FROM {table}
+            WHERE product_id NOT IN (SELECT product_id FROM subscriptions)
+            RETURNING *;""")
+        data = cur.fetchall()
+    conn.commit()
+    logging.info(data)
+    return [dict(i) for i in data]
+
+
+def handler(_event, _context) -> str:
```

Comment:
Stops pylint complaining

```suggestion
def handler(_event, _context) -> str: # pylint: disable=unused-argument
```

Reply:
The underscore already stops pylint from barking